cli/start.go

```diff
@@ -23,17 +23,17 @@ func startRun(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	err = newController.Start(req)
+	taskObject, err := newController.Start(req)
 	if err != nil {
 		fmt.Printf("%#v\n", maskAny(err))
 		os.Exit(1)
 	}
 
-	if req.SliceIDs == nil {
-		fmt.Printf("Started group '%s'\n", req.Group)
-	} else if len(req.SliceIDs) == 0 {
-		fmt.Printf("Started all slices of group '%s'\n", req.Group)
-	} else {
-		fmt.Printf("Started %d slices for group '%s': %v", len(req.SliceIDs), req.Group, req.SliceIDs)
-	}
+	maybeBlockWithFeedback(blockWithFeedbackCtx{
+		Request:    req,
+		Descriptor: "start",
+		NoBlock:    globalFlags.NoBlock,
```

Comment:
I like that the function is not relying on the global flag and instead we pass it as an argument. :cake: